guides/v2.1/frontend-dev-guide/translations/theme_dictionary.md

```diff
@@ -61,6 +61,16 @@ The locale dictionary would use the default values (keys) in the left column fol
 "Card Verification Number",	<translation>
 ```
 
+Translations are applied in following manner:
```

Comment:
How is this order any different than what we say earlier in this topic under the 'How Magento applies locales'?

Reply:
Hello @eriksmarr 

Yes, I agreed My added content was duplicate,  I want to show the clear information to the user how Magento applies translation in a simple way. As per your advise I have removed duplicated content and update my point into existing content.  

Thank you.